tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -181,6 +181,11 @@ def generate_files(list, args):
         files_list.append('<file src=' + '"' + os.path.join(args.ort_build_path, args.build_config,
                                                             'microsoft.ai.machinelearning.winmd') +
                           '" target="lib\\uap10.0\\Microsoft.AI.MachineLearning.winmd" />')
+        # Add custom operator headers
+        files_list.append('<file src=' + '"' +
+                          os.path.join(args.sources_path,
+                                       'onnxruntime\\core\\providers\\dml\\dmlexecutionprovider\\inc\\mloperatorauthor.h') +
```

Comment:
Might be better to match the case of the actual path (onnxruntime\core\providers\dml\ **DmlExecutionProvider\inc\MLOperatorAuthor.h** )

Reply:
Yeah, I actually had it the other way first. Those folders are the few rare folders with capitalization, so I think the right fix is probably to rename those folders to lower case...

Reply:
Are you going to include the folder renaming in this change?